### ISSUE TITLE
Ease local dev while keeping gnoland minimal

### ISSUE THREAD

**Issue by moul** [gnolang/gno#1020]
Currently sharing my screen with @gfanton for onboarding.

My notes that I will translate into a proposal after:

**Comment by waymobetta** [comment]
> adding a second genesis file optional

^ What is this for- so that we don't edit the original which should be locked and no longer editable or some other reason?

> link to new faucet on discord

^ We should probably pin this link rather than needing to share it among the various channels. I'll speak with @Ticojohnny because I haven't found a location where we have shared this faucet link in a single place. We have been sharing faucet links within channels whenever people ask for it which is inefficient.

Or is this in relation to the [Community Faucet Bot](https://github.com/gnolang/community-faucet-bot)?

**Comment by moul** [comment]
> Adding Optional Second Genesis File

Another approach to the second genesis file is using multiple files. This lets some be on git while others are gitignored. 

In my opinion, a better approach could be replacing the single genesis file with a string slice for more flexibility.

> New Faucet Link on Discord

We're finalizing deployment and Discord configuration. Updates soon.

---

@gfanton, please process raw notes to link or create issues. Goal is to address points and close this issue.

**Comment by thehowl** [comment]
@gfanton @moul Do we want to merge this issue with #1604?

**Comment by gfanton** [comment]
@thehowl I agree, waiting for @moul, but I think we can close and link this with #1604
